### DISCUSSION TITLE
AutoMapper 12.0.1 release date?

### DISCUSSION THREAD

**Discussion by bartvanandel** [LuckyPennySoftware/AutoMapper#4200] [NA]
Is there an ETA on the 12.0.1 release? We're having some issues with 12.0.0 that I've just verified as fixed using 12.0.1-preview.0.4, but of course I'd rather not ship a preview version to production unless I really have to.

I can't find any information on how your release plan works (e.g. estimated time between alpha versions, preview, and final release, provided no more blocking issues arise, for instance) on neither your website nor here on GitHub (searched both issues and discussion). Would be nice to have an informational section about that somewhere, even if the general answer is "we don't know, it's done when it's done".

**Comment by jbogard** [comment]
12.0.1 was released yesterday.
